cypress/e2e/profileEdit.cy.ts

```
@@ -0,0 +1,252 @@
+let userFixture: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)